pkgs/applications/networking/instant-messengers/signal-desktop/default.nix

```diff
@@ -123,6 +124,7 @@ in stdenv.mkDerivation rec {
       --prefix LD_LIBRARY_PATH : "${lib.makeLibraryPath [ stdenv.cc.cc ] }"
       ${customLanguageWrapperArgs}
       --add-flags "\''${NIXOS_OZONE_WL:+\''${WAYLAND_DISPLAY:+--enable-features=UseOzonePlatform --ozone-platform=wayland}}"
+      --prefix PATH : ${lib.makeBinPath [xdg-utils]}
```

Comment:
I should be `--suffix` not `--prefix`. I and at least a few others write our own `xdg-open` scripts. When this is `--prefix` we would need to inject our custom `xdg-open` using an overlay and then rebuild all package like this, whereas with `--suffix` we can override at runtime.

Reply:
Sonds good! Didn't think of that, I'll take your word for it.

Just confirmed that it it works all the same for my use-case, so commit pushed!